packages/bind/spec/bindingDependencyBehaviors.ts

```diff
@@ -449,7 +450,7 @@ describe('Binding dependencies', function () {
       expect(vm.getSubscriptionsCount()).toEqual(1);
 
             // remove the element and re-set the view-model; the subscription should be cleared
-      testNode.parentNode.removeChild(testNode);
+      testNode.parentNode?.removeChild(testNode);
```

Comment:
seems odd that `?.` is needed, suggests the test is unstable/flaky (but it may be ok - just noting)

Reply:
i added `expect(testNode.parentElement).not.toBeNull();`